src/components/CodeAssignmentModal/constants.jsx

```diff
@@ -0,0 +1,27 @@
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+import { EMAIL_ADDRESS_TEXT_FORM_DATA, EMAIL_ADDRESS_CSV_FORM_DATA } from '../../data/constants/addUsers';
+import { MODAL_TYPES } from '../EmailTemplateForm/constants';
+import { EMAIL_TEMPLATE_FIELDS } from '../EmailTemplateForm';
+import CheckboxWithTooltip from '../ReduxFormCheckbox/CheckboxWithTooltip';
+
+export const ASSIGNMENT_ERROR_TITLES = {
+  [MODAL_TYPES.assign]: 'Unable to assign codes',
+  [MODAL_TYPES.save]: 'Unable to save template',
+};
+export const EMAIL_TEMPLATE_NUDGE_EMAIL_ID = 'enable-nudge-emails';
+export const textAreaKey = EMAIL_ADDRESS_TEXT_FORM_DATA;
```

Comment:
is there a reason to export a constant of a constant? Honestly just curious why you wouldn't just use the base constant.

Reply:
Heh, yeah, mostly I was feeling like `textAreaKey` might be better name for this file. (And I didn't want to rename everywhere it occurred immediately and forgot to do cleanup. I'll do some cleanup here.